src/Home.css

```diff
@@ -118,11 +118,21 @@
   padding: 0;
 }
 
+.darkMode {
+  padding: 0 32px;
+  margin-top: auto;
+  margin-left: auto;
+}
+
```

Comment:
If this style is for the `DarkMode` component, I would add it to that component.